src/System.Private.CoreLib/shared/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -266,6 +267,8 @@ internal DateTimeFormatInfo(CultureData cultureData, Calendar cal)
             _cultureData = cultureData;
 
             Calendar = cal;
+            // workaround compiler issue
```

Comment:
We have a note in some issue to remove this i assume

Reply:
now we do: https://github.com/dotnet/corefx/issues/36427#issuecomment-477777088